assets/js/blocks/reviews-by-product/editor-block.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { debounce } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getOrderArgs, getReviews } from './utils';
+import LoadMoreButton from '../../frontend/components/load-more-button';
+import OrderBySelect from '../../frontend/components/order-by-select';
+import Reviews from '../../frontend/components/reviews';
+import withComponentId from '../../frontend/hocs/with-component-id';
+
+/**
+ * Block rendered in the editor.
+ */
+class EditorBlock extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			reviews: [],
+			totalReviews: 0,
+		};
+
+		this.debouncedLoadFirstReviews = debounce( this.loadFirstReviews.bind( this ), 400 );
+	}
+
+	componentDidMount() {
+		this.loadFirstReviews();
+	}
+
+	componentDidUpdate( prevProps ) {
+		if (
+			prevProps.attributes.orderby !== this.props.attributes.orderby ||
+			prevProps.attributes.productId !== this.props.attributes.productId ||
+      prevProps.attributes.reviewsOnPageLoad !== this.props.attributes.reviewsOnPageLoad
```

Comment:
Seems to be some indentation issues here (code style).

Reply:
Fixed in 741d04ab0919f8392ff5c5ddeb637911aadd4e9e.